src/python/aqueduct_executor/operators/connectors/tabular/extract.py

```diff
@@ -1,13 +1,88 @@
-from typing import Any, Optional, Union
-
+import re
+from typing import Any, Optional, Union, List, Dict
+from datetime import date
 from aqueduct_executor.operators.connectors.tabular import common, models
+from aqueduct_executor.operators.utils import constants
+
+# Regular Expression that matches any substring appearance with
+# "{{ }}" and a word inside with optional space in front or after
+# Potential Matches: "{{today}}", "{{ today  }}""
+#
+# Duplicated in the SDK at `sdk/aqueduct/integrations/sql_integration.py`.
+# Make sure the two are in sync.
+TAG_PATTERN = r"{{\s*\w+\s*}}"
```

Comment:
not an regex expert, does this allow some special characters like underscore and dashes as parameter name?

Reply:
ah this actually misses dashes. Will fix.

Reply:
It does not intend to catch spaces btw (eg. `{{ hello world }}` will not be matched)

Reply:
@likawind the current regex `\w+` matches letters,numbers, and underscores but not dash `-` I believe. Do you think it is necessary to add that in?